libraries/AP_GPS/AP_GPS.h

```diff
@@ -157,9 +174,8 @@ class AP_GPS
         return status(primary_instance);
     }
 
-    // Query the highest status this GPS supports
+    // Query the highest status this GPS supports (always reports GPS_OK_FIX_3D for the blended GPS)
```

Comment:
We don't have to do it by any means, I'm just trying to hold for as consistent an API without having to read all the special cases from the header file every time :)